ui/components/ui/token-currency-display/token-currency-display.stories.tsx

```diff
@@ -1,9 +1,8 @@
 import React from 'react';
-import { Meta, Story } from '@storybook/react';
```

Comment:
Updates story to CSF3.0 format

![Screenshot 2024-10-23 at 3 39 23 PM](https://github.com/user-attachments/assets/706bb7f7-22a6-49c6-8fcb-834e488fc9fc)